core/src/test/java/org/apache/druid/math/expr/ExprEvalTest.java

```diff
@@ -94,8 +94,8 @@ public void testStringArraySerdeToBig()
     expectedException.expect(ISE.class);
     expectedException.expectMessage(StringUtils.format(
         "Unable to serialize [%s], size [%s] is larger than max [%s]",
-        ExprType.STRING_ARRAY,
-        14,
+        ExpressionType.STRING_ARRAY,
+        15,
```

Comment:
these changes in values are because arrays are stored with an extra byte now (one for `ARRAY`, one for element type).